index.js

```diff
@@ -0,0 +1,46 @@
+document.addEventListener("DOMContentLoaded", () => {
+    let div = document.querySelector("div")
+    let form = document.querySelector("form")
+    let review = document.querySelector("#review")
+    let submitReview = document.querySelector("#submitReview")
+    let ul = document.querySelector("ul")
+
+    form.addEventListener("submit", (event) => {
+        event.preventDefault()
+        let li = document.createElement("li")
+        li.innerText = review.value
+        review.appendChild(li)
+        ul.appendChild(li)
+        searchFilm(review.value, submitReview.value)
+    })
+
+    const searchFilm = async () => {
+        try {
+            let res = await axois.get(`https://ghibliapi.herokuapp.com/films`)
+            chooseFilm(res.data)
+        } catch (error) {
+    
+        }
+    }
+
+    const chooseFilm = async () => {
+        let selectFilm = document.querySelector("#selectFilm")
+        selectFilm.innerText = film.title
```

Comment:
In here you would want to use the value of the selected `option` tag to select which movie you show.